packages/docs/src/routes/community/media/index.tsx

```diff
@@ -222,7 +222,7 @@ export const Section = component$(
 export default component$(() => {
   useStyles$(styles);
   return (
-    <article class="media">
+    <article class="media media-section">
```

Comment:
why was this extra class needed? 🤔

Reply:
Because the `docs.css` is overwriting the styles. Not sure if that is the best solution.